packages/consts/src/consts.ts

```diff
@@ -144,7 +145,15 @@ export const DNS_SAFE_NAME_MAX_LENGTH = 63;
 export const ACTOR_NAME = {
     MIN_LENGTH: 3,
     MAX_LENGTH: DNS_SAFE_NAME_MAX_LENGTH, // DNS-safe string length
-    REGEX: DNS_SAFE_NAME_REGEX,
+    REGEX: /^(?!.*apify)([a-z0-9]|[a-z0-9][a-z0-9-]*[a-z0-9])$/i,
+    ADMIN_REGEX: DNS_SAFE_NAME_REGEX,
```

Comment:
@mtrunkat If I add the min and max length in the DNS_SAFE_NAME_REGEX, it might potentially throw weird errors in forms because in some cases, it is only necessary to match the expression but not the length.

Example: https://github.com/apify/apify-core/blob/develop/src/packages/schemas/src/key_value_stores.ts#L9-L15

Should I add the length there?

Reply:
The advantage of separating `DNS_SAFE_NAME_MAX_LENGTH` and `DNS_SAFE_NAME_REGEX` is that you get better error messages: Xxx is too long vs. Xxx doesn't match regular expression